configs/spacy.json

```diff
@@ -9,28 +9,26 @@
   ],
   "selectors": {
     "lvl0": {
-      "selector": "//*[contains(@class,'c-sidebar')]//*[contains(@class,'is-active')][1]/preceding::*[contains(@class,'u-color-dark')][1]",
+      "selector": "//*[contains(@class,'sidebar')]//*[contains(@class,'is-active')][1]/preceding::*[contains(@class,'.h0')][1]",
       "type": "xpath",
       "global": true,
       "default_value": "Documentation"
     },
-    "lvl1": {
-      "selector": "//article//h1/text()",
-      "type": "xpath"
-    },
-    "lvl2": "article h2",
-    "lvl3": "article h3",
-    "lvl4": "article h4, article table td:first-child, .c-table__row--foot td:nth-child(2)",
```

Comment:
`.c-table__row--foot td:nth-child(2)` is usually the return type of the documented function/method, which isn't very relevant in our case. So I've removed that.